pypesto/C.py

```diff
@@ -275,13 +275,6 @@ class InnerParameterType(str, Enum):
 CSV = 'csv'  # return file format
 H5 = 'h5'  # return file format
 
-
-###############################################################################
-# SELECT
-
-TYPE_POSTPROCESSOR = Callable[["ModelProblem"], None]  # noqa: F821
```

Comment:
@dilpath : Had to shuffle some things around to break circular dependencies. Is that okay?

Reply:
Looks fine to me, thanks :+1: